src/id/mangatale/src/eu/kanade/tachiyomi/extension/id/mangatale/Ikiru.kt

```diff
@@ -129,7 +129,7 @@ class Ikiru : ParsedHttpSource() {
 
         val chapterListUrl = "$baseUrl/ajax-call".toHttpUrl().newBuilder()
             .addQueryParameter("manga_id", mangaId)
-            .addQueryParameter("page", "${Random.nextInt(5, 99)}") // keep above 3 for loading hidden chapter
+            .addQueryParameter("page", "${Random.nextInt(5, 9999)}") // keep above 3 for loading hidden chapter
```

Comment:
If they are blocking certain numbers, just pick something outside of that range
```suggestion
            .addQueryParameter("page", "${Random.nextInt(99, 9999)}") // keep above 3 for loading hidden chapter
```

Reply:
so far only noticed single number but sure